src/main/java/gregtech/client/utils/BloomEffectUtil.java

```diff
@@ -29,14 +30,16 @@ public class BloomEffectUtil {
     public static BlockRenderLayer BLOOM;
     private static Framebuffer BLOOM_FBO;
     private static Map<IBloomRenderFast, List<Consumer<BufferBuilder>>> RENDER_FAST;
+    private static boolean nothiriumLoaded;
 
-    public static BlockRenderLayer getRealBloomLayer(){
+    public static BlockRenderLayer getRealBloomLayer() {
         return Shaders.isOptiFineShaderPackLoaded() ? BlockRenderLayer.CUTOUT : BLOOM;
     }
 
     public static void init() {
         BLOOM = EnumHelper.addEnum(BlockRenderLayer.class, "BLOOM", new Class[]{String.class}, "Bloom");
         RENDER_FAST = Maps.newHashMap();
+        nothiriumLoaded = Loader.isModLoaded("nothirium");
```

Comment:
I think it would be nice for code cleanliness to keep this boolean in a function in `Shaders` like the optifine check